core/templates/pages/exploration-editor-page/services/exploration-rights.service.spec.ts

```diff
@@ -279,6 +279,19 @@ describe('Exploration rights service', function() {
       sampleDataResults.rights.viewable_if_private);
   });
 
+  it('should callback to be called in saveModeratorChangeToBackend',
+    function() {
+      $httpBackend.expectPUT('/createhandler/moderatorrights/12345').respond(
+        200, sampleDataResults);
+      const onUnpublishingcallbackFunc =
+        jasmine.createSpy('onUnpublishingcallback');
+      ers.saveModeratorChangeToBackend(
```

Comment:
Add new line above? since the L288-L290 is an action. But in the previous test it is not split like that.

Reply:
Added new line DONE